shoop/admin/modules/orders/static_src/create/reducers/lines.js

```diff
@@ -54,15 +54,15 @@ function getDiscountsAndTotal(quantity, baseUnitPrice, unitPrice, updateUnitPric
     if (updateUnitPrice) {
         updates.unitPrice = unitPrice;
     }
-    var totalBeforeDiscount = baseUnitPrice * quantity;
+    var totalBeforeDiscount = (baseUnitPrice * quantity).toFixed(2);
     var total = +(unitPrice * quantity).toFixed(2);
     updates.total = total;
     if (baseUnitPrice < unitPrice || unitPrice < 0) {
         updates.discountPercent = 0;
         updates.discountAmount = 0;
         return updates;
     }
-    var discountAmount = totalBeforeDiscount - total;
+    var discountAmount = (totalBeforeDiscount - total).toFixed(2);
```

Comment:
The precision shouldn't be cut to 2 decimals, since line prices should use higher precision than currency's precision. It seems that the total is also cut to 2 decimals (in line 58), but that is incorrect too. Would it work better if you just remove that?

Order total should be rounded to currency's precision though. If that's not already done in JS, then it should be implemented.

Also, we eventually want currency's precision to be configurable too. So if the 2 decimals really needs to be hardcoded somewhere, please add `SHOOP-1912` tag to those lines.